docs/sources/tempo/configuration/grafana-alloy/tail-sampling/policies-strategies.md

```diff
@@ -0,0 +1,407 @@
+Generally, composite policies end with an `always_sample` policy type to ensure that traces are still sampled should none of the other aggregated policies inside it match.
+
+Refer to the [`composite`](https://grafana.com/docs/alloy/latest/reference/components/otelcol/otelcol.processor.tail_sampling/#composite-block) documentation for more information.
+
+```alloy
+policy {
+    // The example below is for the `composite` sampling policy
+    name = "composite-policy"
+    type = "composite"
+    composite {
+            // Limit sampling to a maximum of 500 spans per second.
+            max_total_spans_per_second = 500
+            // Evaluate the policies in the following order.
+            // This acts like the default drop-through policy matcher.
+            policy_order = ["composite-policy-keyvalue", "composite-policy-always"]
+            // Sample any trace with a 20x status code.
+            composite_sub_policy {
+                name = "composite-policy-keyvalue"
+                type = "string_attribute"
+                string_attribute {
+                    key = "http.code"
+                    enabled_regex_matching = true
+                    values = [ "20[0|1|2]" ]
+                }
+            }
+            // Finalise with an `always_sample` policy type.
+            composite_sub_policy {
+                name = "composite-policy-always"
+                type = "always_sample"
+            }
+            // Allocate 80% of sampling to the `string_attribute` policy.
+            // The remaining 20% is allocated to the `always_sample` policy.
+            rate_allocation {
+                policy = "composite-policy-keyvalue"
+                percent = 80
+            }
+            rate_allocation {
+                policy = "composite-policy-always"
+                percent = 20
+            }
+        }
+    }
+```
+
+### Latency
+
+The latency policy is used to look for slow-running spans. For example, you can use it for performance monitoring. Slow-running spans can indicate performance bottlenecks. In the example below, we are sampling for traces that are between 3s and 10s long.
+
+Refer to the <code>[latency](https://grafana.com/docs/alloy/latest/reference/components/otelcol/otelcol.processor.tail_sampling/#latency)</code> documentation for more information.
+
+```alloy
+policy {
+    // The example below is for the `latency` sampling policy
+    name = "example_latency"
+    type = "latency"
+
+    latency {
+      // The minimum duration for a trace to be sampled
+      threshold_ms = 3000
+      // The maximum duration for a trace to be sampled
+      max_duration_ms = 10000
+    }
+  }
+```
+
+### Numeric attribute
+
+The `numeric_attribute` policy lets you sample based on a number of attributes. In this example below, spans are sampled where `http.response_content_length` is between 10,000 and 500,000. The sample span below would be captured because the attribute value falls within the specified range.
```

Comment:
```suggestion
The `numeric_attribute` policy lets you sample based on a number of attributes. 
In this example below, spans are sampled where `http.response_content_length` is between 10,000 and 500,000. 
The sample span below would be captured because the attribute value falls within the specified range.
```